dat/events/dvaered/warlords_battle.lua

```diff
@@ -2,7 +2,7 @@
 <?xml version='1.0' encoding='utf8'?>
 <event name="Warlords battle">
   <trigger>enter</trigger>
-  <chance>5</chance>
+  <chance>100</chance>
```

Comment:
Probably left in the 100% chance when debugging

Reply:
Pfff... I should never push early in the morning.